### PR TITLE
Fix constructor signature

### PR DESCRIPTION
**Pull request by gokhanbarisaker**
Resolves: #87

### CODE REVIEW

**Comment by paddybyers** [comment]
LGTM

File: lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java [hunk 1]
```diff
@@ -673,8 +673,8 @@ public Object getTokenRequest(TokenParams params) throws AblyException {
 			/* Publish a message */
 			Message messagePublishee = new Message(
 					"wildcard",	// name
-					"brian that is called brian",	// clientId
-					String.valueOf(System.currentTimeMillis())	// data
+					String.valueOf(System.currentTimeMillis()), "brian that is called brian"    // clientId
+					// data
 			);
```
Comment (mattheworiordan) [inline lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java]: This looks wrong, given the signature is `name, data, clientId`, then your comments of `// name, //clientId` is wrong?  How did this pass if so or have I misread this?

Comment (gokhanbarisaker) [inline lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java]: The signature and usage is right. I should fix the comments for them

File: lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java [hunk 2]
```diff
@@ -673,8 +674,8 @@ public Object getTokenRequest(TokenParams params) throws AblyException {
 			/* Publish a message */
 			Message messagePublishee = new Message(
 					"wildcard",	// name
-					"brian that is called brian",	// clientId
```
Comment (paddybyers) [inline lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java]: Different comment style from above :(

**Comment by paddybyers** [comment]
Otherwise LGTM

**Comment by mattheworiordan** [comment]
Small comment from @paddybyers but otherwise good

**Comment by gokhanbarisaker** [comment]
Done

**Comment by mattheworiordan** [comment]
LGTM

**Pull request merged**